packages/suite/src/components/wallet/AccountException/index.tsx

```diff
@@ -24,20 +25,19 @@ const Title = styled(H2)`
     color: ${props => props.theme.TYPE_DARK_GREY};
 `;
 
-interface Props {
-    account: SelectedAccountException;
-}
-
-const getExceptionPage = ({ account }: Props) => {
-    switch (account.loader) {
+const getExceptionPage = (
+    loader: SelectedAccountException['loader'],
+    network: SelectedAccountException['network'],
+) => {
+    switch (loader) {
         case 'auth-failed':
             return <AuthFailed />;
         case 'discovery-error':
             return <DiscoveryFailed />;
         case 'discovery-empty':
             return <DiscoveryEmpty />;
         case 'account-not-enabled':
-            return <AccountNotEnabled network={account.network} />;
+            return <AccountNotEnabled network={network as Network} />;
```

Comment:
Please try to avoid using `as` type assertion whenever it's possible. Here  I believe it's much better to handle undefined state. 

FYI: if somewhere you are really sure that something can't be undefined and it's only typescript is not able to see it, you could use just `!`  (like `network!` but don't use it here, please)

Reply:
I was trying to avoid it but it only can be done by passing the whole `SelectedAccountException` object and not the 2 fields… TS is not smart enough to narrow unions in switches :( The `network` will be present there because it is not optional when loader is `“account-not-enabled”`